src/core/parser.zsh

```diff
@@ -6,18 +6,10 @@ _fzf_complete_get_command_pos() {
     echo ${arguments[(i)$cmd]}
 }
 
-_fzf_complete_trim_env() {
-    local command_pos=$1
-    shift 1
-    local arguments=("${(Q)${(z)@}[@]}")
-    echo ${(q)arguments[$command_pos, -1]}
-}
-
-_fzf_complete_get_env() {
-    local command_pos=$1
-    shift 1
-    local arguments=("${${(z)@}[@]}")
-    echo ${arguments[1, $command_pos - 1]}
```

Comment:
Do you need to remove those functions? I think that doing `[pos, -1]` or `[1, pos -1]` to trim/get env each time in the head of the complete functions is redundant and difficult to understand.

Reply:
Actually, no. However, now that the logic is exactly the same when retrieving env part and command+argument part, it is not as difficult to understand as it was before, isn't it?